source/core/VirtualInstrument.cpp

```diff
@@ -34,7 +34,7 @@ NIError VirtualInstrument::Init(TypeManagerRef tm, Int32 clumpCount, TypeRef par
     _locals->SetElementType(localsType, false);
     _clumps->Resize1D(clumpCount);
     _lineNumberBase = lineNumberBase;
-    _clumpSource = clumpSource;
```

Comment:
To clarify, it's because there was a constructor which took a SubString pointer, but no operator=.  The compiler was synthesizing an operator= out of the constructor, but with the explicit keyword added, it would no longer do so.